skimage/metrics/_contingency_table.py

```diff
@@ -4,7 +4,7 @@
 __all__ = ['contingency_table']
 
 
-def contingency_table(im_true, im_test, ignore_labels=[], normalize=False):
+def contingency_table(im_true, im_test, *, ignore_labels=[], normalize=False):
```

Comment:
Wait isn't using mutable default values bad practice? Doesn't seem like an issue for this particular function but better safe than sorry?

Reply:
Yep, need to fix this, good catch!